examples/aws-lambda-okta-webhook/terraform-aws-okta-webhook/src/index.ts

```diff
@@ -1,5 +1,5 @@
-import { OktaGroupIntegration, OktaUserIntegration } from '@indent/okta-webhook'
-import { getLambdaHandler } from '@indent/runtime-aws'
```

Comment:
Both of these reflect the new package names we're using elsewhere in the repo. Are these packages also uploaded to npm with the new name?

Reply:
those are references to the old naming convention while experimenting with the package structure. New ✅ convention is: `@indent/integration-okta` and `@indent/runtime-aws-lambda`